lib/utils.js

```diff
@@ -71,34 +71,22 @@ module.exports.parseCommits = async ({ logs, verbose = false }) => {
 }
 
 module.exports.formatNotes = ({ parsed, version, previous }) => {
-  // group by T1 & T2, enrich msg with label
+  // group by T1, enrich msg with label
   const byT1 = parsed.reduce((acc, { t1, labels, msg }) => {
-    // to organize commits without T2
-    const t2 = msg['t2'] ? msg.t2 : 'others'
-    delete msg.t2
-    acc[t1] = {
-      ...acc[t1],
-      [t2]: [...(acc[t1] ? acc[t1][t2] || [] : []), { ...msg, labels }],
-    }
+    acc[t1] = [...(acc[t1] || []), { ...msg, labels }]
```

Comment:
actually i was talking more about labels (the NLP detected ones) when i mentioned 

> we should probably also drop the secondary label and only show the first to be less confusing

:trollface:

Reply:
🤦‍♀️ 😅